### ISSUE TITLE
"Combinations" just crashes the app

### ISSUE THREAD

**Issue by LunaUrsa** [TripSit/tripmobile#49]

**Comment by LunaUrsa** [comment]
Yeah despite all that error message it seems like it came down to the HTTPS again: https://github.com/TripSit/tripmobile/commit/d051eeaaaa0d7c2620729dea8d306312db280248

**Comment by tacerus** [comment]
Other users reported an error message instead of a crash AFAIK. Maybe your system handled it differently.

**Comment by unresolvedsymbol** [comment]
Also if it doesn't crash it just provides useful information, ex: https://a.pomf.cat/mgdrwk.png
